static/scss/calculator.scss

```diff
@@ -81,3 +81,56 @@
     }
   }
 }
+
+@include breakpoint(phone) {
+  .financial-aid-calculator, .confirm-dialog {
+
+    .salary-input, .salary-input > div {
+      display: block;
+    }
+
+    h3 {
+      border-bottom: none !important;
+      padding: 20px 20px 10px !important;
+    }
+
+    .financial-aid-calculator-body {
+      overflow: scroll !important;
+      padding: 0 20px 10px !important;
```

Comment:
there's a lot of `!important` here, is there any way we can avoid these? I know some things are styles set by mdl which we have to override, but it would be good to avoid `!important` as much as possible.